File: Modules/ServerTypes/LeaveApplication/LeaveTypes.ts
import { Decorators } from "@serenity-is/corelib";

export enum LeaveTypes {
    Full = 0,
    Half = 1
}
Decorators.registerEnumType(LeaveTypes, 'HRMSoftware.LeaveApplication.LeaveTypes', 'LeaveApplication.LeaveTypes');

File: Modules/HumanResource/LeaveApplication/LeaveApplication/LeaveTypes.cs
using System.ComponentModel;

namespace HRMSoftware.LeaveApplication;
[Serenity.ComponentModel.EnumKey("LeaveApplication.LeaveTypes")]
public enum LeaveTypes
{
    [Description("Full Day Leave")]
    Full = 0,
    [Description("Half Day Leave")]
    Half = 1
}